qiskit_neko/tests/machine_learning/test_neural_networks_primitives.py

```diff
@@ -16,7 +16,6 @@
 from ddt import ddt, data, unpack
 from qiskit import QuantumCircuit
 from qiskit.circuit import Parameter
-from qiskit.primitives import Sampler as ReferenceSampler, Estimator as ReferenceEstimator
```

Comment:
Same comment as above, ideally we could simply import `StatevectorSampler` and `StatevectorEstimator` as Referece and the code should work.

Reply:
Done in: https://github.com/Qiskit/qiskit-neko/pull/50/commits/74867835b17efbd4646526a109a521dc77cce678